defaults/main.yml

```diff
@@ -1,12 +1,23 @@
 ---
+openio_kibana_namespace: "OPENIO"
+openio_kibana_serviceid: "0"
+
+openio_kibana_gridinit_dir: "/etc/gridinit.d"
+openio_kibana_gridinit_file_prefix: "{{ openio_kibana_namespace }}-"
+
+openio_kibana_sysconfig_dir: "/etc/oio/sds/{{ openio_kibana_namespace }}/{{ openio_kibana_servicename }}"
+openio_kibana_log_dir: "/var/log/oio/sds/{{ openio_kibana_namespace }}/{{ openio_kibana_servicename }}"
+openio_kibana_pid_dir: "/run/oio/sds/{{ openio_kibana_namespace }}"
```

Comment:
`openio_kibana_pid_directory` like in other roles

Reply:
so `sysconfig`, `log` are suffixed with `_dir` but `pid` is suffixed with `_directory` 

where is the consistency here ?

i'll fix this for those 3 EFK roles but this does not make sens globally IMHO

Reply:
fixed